mixed/spark/v3.2/spark/pom.xml

```diff
@@ -101,6 +101,14 @@
             <version>${scala.version}</version>
         </dependency>
 
+        <dependency>
+            <!-- Scala module 2.12.3 requires Jackson Databind version >= 2.12.0 and < 2.13.0 -->
```

Comment:
It seems to be that the scala version here is 2.12.15.

Reply:
Spark 3.2 means using 2.12.3, so it requires Jackson version [2.12.0, 2.13.0) , the comment here want to record why do we need to add a provided dependency here